src/main/kotlin/lotto/domain/Lotto.kt

```diff
@@ -0,0 +1,22 @@
+package lotto.domain
+
+class Lotto private constructor(val lottoNumbers: Set<LottoNumber>) {
+
+    init {
+        val lottoNumbersSize = lottoNumbers.size
+        require(lottoNumbersSize == LOTTO_SIZE)
+    }
+
+    constructor() : this(LottoPreset.shuffled().take(6).sortedBy { it.number }.toSet())
+
+    constructor(vararg number: Int) : this(number.map { LottoNumber(it) }.toSet())
```

Comment:
saso1 

이러한 구문을 reference로 넘길 수 있습니다.
```suggestion
    constructor(vararg number: Int) : this(number.map(::LottoNumber).toSet())
```
lambda로 작성하는 것과 어떤 차이가 있는지 참고할 수 있는 글을 소개해드릴게요 :slightly_smiling_face:
https://proandroiddev.com/kotlin-lambda-vs-method-reference-fdbd175f6845

참고로 저는 가능한 곳에서는 꼭 함수 reference를 활용하는 것을 지향합니다. lambda로 가져온 데이터를 별도로 조작하지 않고 그대로 함수 시그니처로 넘긴다는 것을 드러낼 수 있기 때문입니다.

Reply:
f6e9af8e53af74e374b3381f918a6b321ff960f0